guides/common/modules/proc_configuring-oauth.adoc

```diff
@@ -0,0 +1,29 @@
+[id="configuring-oauth"]
+= Configuring OAuth
+
+Use `{foreman-installer}` to change OAuth settings on {ProjectServer}.
+Enter the following command to list all OAuth-related installer options:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {foreman-installer} --full-help | grep oauth
+----
+
+.Enabling OAuth user mapping
+
+By default, {Project} authorizes all OAuth API requests as the built-in anonymous API administrator account.
+Therefore, API responses include all {Project} data.
+However, you can also specify the {Project} user that makes the request and restrict access to data to that user.
+
+To enable OAuth user mapping, enter the following command:
+
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {foreman-installer} --foreman-oauth-map-users true
+----
```

Comment:
IIRC deploying a new content proxy relies on oauth and I'm not sure what enabling user mapping does to it, if anything. @ekohl could you please chime in?